.github/workflows/lint.yml

```
@@ -4,8 +4,17 @@ on: [push, pull_request, workflow_dispatch]
 
 jobs:
   pre-commit:
+    name: Run Pre-Commit
     runs-on: ubuntu-latest
+
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-python@v2
-    - uses: pre-commit/action@v2.0.0
+    - name: Checkout repo
```

Like "set up"/"setup" below, "checkout" is a noun, "check out" is the verb form:
```suggestion
    - name: Check out repo
```

---

Yeah, I thought about that and almost changed it to "Check out" here, but I didn't for some reason. Thanks.

I can commit this, but I'll wait on a decision on this PR first; if we do have to split it up, I'd have to manually `rebase -i` it in.